public/student.html

```diff
@@ -306,12 +309,12 @@ <h1 class="hyphenate" id="aqua">Aquanette Sanders</h1>
       </div>
       <div class="col-lg-6  pl-lg-5 pl-1 pt-lg-0 pt-2 py-5 d-flex flex-column justify-content-lg-around">
          <div class>
-            <h1 class="hyphenate" id="alina">Alina Spera <span style="font-size: 1rem;">(she/her)</span></h1>
+            <h3 class="h1 hyphenate" id="alina">Alina Spera <span style="font-size: 1rem;">(she/her)</span></h3>
             <p class="lead border-dark border-bottom">The University of Texas at El Paso, Ecology and Evolutionary Biology</p>
             <p>I am a New Jersey native with educational and research background in marine science and biology. My research interests include ecology and biogeochemistry with a focus on studying rapidly changing coastal systems. My project as part of the BLE integrates high-frequency environmental sensor data with carbon chemistry to model and understand biogeochemical and ecological processes occurring in the Beaufort lagoons and nearby aquatic systems.</p>
             <p>I am the current BLE student representative, advocating for the interests of all BLE-affiliated students.</p>
             <button class="btn btn-outline-primary m-1"><a href="mailto:aspera@miners.utep.edu"><i class="  far fa-paper-plane"></i>  aspera@miners.utep.edu</a></button>
-            <button class="btn btn-outline-primary m-1"><a href="https://orcid.org/0000-0001-5025-1260" target="_blank">Publications</a></button>
+            <button class="btn btn-outline-primary m-1"><a href="https://orcid.org/0000-0001-5025-1260" target="_blank" aria-label="open page in new tab">Publications</a></button>
```

Comment:
specify what you're opening in new tab

Reply:
fixed in c4d830a7a3cf15b16021792b9e4da7d42427b8c5